app/app.iml

```diff
@@ -95,5 +95,11 @@
     </content>
     <orderEntry type="jdk" jdkName="Android API 15 Platform" jdkType="Android SDK" />
     <orderEntry type="sourceFolder" forTests="false" />
```

Comment:
Where there previously no tests for this project? You have added the dependencies necessary for testing?